.github/CODEOWNERS

```diff
@@ -180,6 +180,8 @@
 
 
 # Global frontend services
+/core/templates/dev/head/services/LoggerService.ts @showtim3
+/core/templates/dev/head/services/EventService.ts @showtim3
```

Comment:
Swap this with `LoggerService.ts`, this is to keep the list alphabetized.

Reply:
Done